core/src/main/scala/com/karumi/shot/domain/ShotFolder.scala

```diff
@@ -7,25 +7,26 @@ case class ShotFolder(
     private val buildFolderPath: FilePath,
     private val buildType: String,
     private val flavor: Option[String],
-    private val directorySuffix: Option[String]
+    private val directorySuffix: Option[String],
+    private val separator: String
```

Comment:
Adding separator in ShotFolder is necessary due to in moveComposeScreenshotsToRegularScreenshotsFolder 145 line there is replace fragment of file's path. It didn't work on Windows according to different slash.